frontend/simple/js/state.js

```diff
@@ -135,6 +188,7 @@ const actions = {
       }
       // TODO: verify each entry is signed by a group member
       await db.addLogEntry(contractId, entry)
+      entry.data.hash = hash
```

Comment:
Why is this being done? The entry should never be modified by an external entity like this. We should probably figure out a way to enforce that by doing the JS equivalent of marking everything in there `@private`.